backend/import/test_mo_api.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+import mo_api
+import do_import
+
+
+def my_mo_get(url):
+    url_file = '.test_data/mo/' + url.replace('/', '#') + '.json'
```

Comment:
I can't help but find this is semi-evil — using `requests_mock` seems simpler, and I'd probably elect to combine the various fixtures into one. That's a matter of taste, though, not a complain _as such_…

Reply:
We'll get to that in a later iteration, I think this works for now.